crates/ruff_linter/src/rules/flynt/helpers.rs

```diff
@@ -1,25 +1,23 @@
 use ruff_python_ast::{self as ast, Arguments, Constant, ConversionFlag, Expr};
 use ruff_text_size::TextRange;
 
-/// Wrap an expression in a `FormattedValue` with no special formatting.
-fn to_formatted_value_expr(inner: &Expr) -> Expr {
-    let node = ast::ExprFormattedValue {
-        value: Box::new(inner.clone()),
+/// Wrap an expression in a `FStringExpressionElement` with no special formatting.
+fn to_formatted_value_expr(inner: &Expr) -> ast::FStringElement {
+    ast::FStringElement::Expression(ast::FStringExpressionElement {
+        expression: Box::new(inner.clone()),
         debug_text: None,
         conversion: ConversionFlag::None,
-        format_spec: None,
+        format_spec: vec![],
         range: TextRange::default(),
-    };
-    node.into()
+    })
 }
 
 /// Convert a string to a constant string expression.
-pub(super) fn to_constant_string(s: &str) -> Expr {
-    let node = ast::ExprConstant {
-        value: s.to_owned().into(),
+pub(super) fn to_constant_string(s: &str) -> ast::FStringElement {
+    ast::FStringElement::Literal(ast::FStringLiteralElement {
```

Comment:
The previous implementation would create a constant string node but it's changed to creating a f-string literal element. Is that an expected behavior? If so, could you change the function name and documentation to reflect that?

Reply:
yes it's expected, this is (and was) only used to build an f-string. makes sense to update the name